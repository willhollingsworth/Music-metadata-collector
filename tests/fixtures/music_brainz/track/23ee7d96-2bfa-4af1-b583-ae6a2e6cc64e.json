{
  "first-release-date": "2022-02-25",
  "disambiguation": "",
  "releases": [
    {
      "packaging": "None",
      "packaging-id": "119eba76-b343-3e02-a292-f0f00644bb9b",
      "text-representation": {
        "script": "Latn",
        "language": "eng"
      },
      "barcode": null,
      "status": "Official",
      "date": "2022-02-25",
      "country": "XW",
      "artist-credit": [
        {
          "name": "Romain Garcia",
          "artist": {
            "sort-name": "Garcia, Romain",
            "disambiguation": "",
            "type-id": "b6e035f4-3ce9-331c-97df-83397230b0df",
            "type": "Person",
            "country": "FR",
            "id": "32c1c2f5-b510-4b57-bdf9-e0a3d7c79857",
            "name": "Romain Garcia"
          },
          "joinphrase": ""
        }
      ],
      "release-events": [
        {
          "area": {
            "sort-name": "[Worldwide]",
            "disambiguation": "",
            "type-id": null,
            "type": null,
            "id": "525d4e18-3d00-31b9-a58b-a146a916de8f",
            "iso-3166-1-codes": [
              "XW"
            ],
            "name": "[Worldwide]"
          },
          "date": "2022-02-25"
        }
      ],
      "quality": "normal",
      "id": "8cc18a30-cac7-4794-bcac-24feebdcda17",
      "status-id": "4e304316-386d-3409-af2e-78857eec5cfe",
      "title": "Before You Go",
      "disambiguation": ""
    }
  ],
  "video": false,
  "artist-credit": [
    {
      "joinphrase": "",
      "artist": {
        "id": "32c1c2f5-b510-4b57-bdf9-e0a3d7c79857",
        "name": "Romain Garcia",
        "sort-name": "Garcia, Romain",
        "disambiguation": "",
        "type-id": "b6e035f4-3ce9-331c-97df-83397230b0df",
        "country": "FR",
        "type": "Person"
      },
      "name": "Romain Garcia"
    }
  ],
  "id": "23ee7d96-2bfa-4af1-b583-ae6a2e6cc64e",
  "length": 241000,
  "title": "Before You Go"
}