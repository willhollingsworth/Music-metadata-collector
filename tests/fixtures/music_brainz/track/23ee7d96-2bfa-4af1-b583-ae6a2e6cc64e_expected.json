{
  "track_name": "Before You Go",
  "track_id": "23ee7d96-2bfa-4af1-b583-ae6a2e6cc64e",
  "artist_name": "Romain Garcia",
  "artist_id": "32c1c2f5-b510-4b57-bdf9-e0a3d7c79857",
  "album_name": "Before You Go",
  "album_id": "8cc18a30-cac7-4794-bcac-24feebdcda17"
}